docs/current/sdk/nodejs/snippets/nodejs-ci/Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+pipeline {
+  agent { label 'dagger' }
+
+  stages {
+    stage("dagger") {
+      steps {
+        sh '''
+            npm ci
+            node index.mjs
```

Comment:
For the other SDKs, we use `main.*` as the script name, should we follow that pattern here and call this `main.mjs`?

Reply:
good idea. Will change in hero on webpage as well.

Reply:
nit: Like I said in the PR, using index.mjs is actually the common naming. main.mjs feels uncommon IMO